webapp/lib/app/utils.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:katikati_ui_lib/components/nav/button_links.dart';
+
+var navLinks = [
+  Link("Dashboard", "/"),
+  Link("Conversations", "/converse/index.html"),
+  Link("Messages", "/configure/messages.html"),
+  Link("Tags", "/configure/tags.html")
```

Comment:
can we add a link to the explorer here as well?
```suggestion
  Link("Tags", "/configure/tags.html"),
  Link("Explore", /explore"),
```